mhs/mhs_common/Pipfile

```diff
@@ -0,0 +1,25 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+coverage = "~=4.5"
+tornado = "*"
+unittest-xml-reporting = "~=2.5"
+
+[packages]
+mhs-common = {editable = true,path = "."}
+defusedxml = "~=0.6"
+integration-adaptors-common = {editable = true,path = "./../../common"}
+aioboto3 = "*"
```

Comment:
From what I understand since we're running all the unit tests through a pip environment the packages need duplicate requirements since they're tested with pipenv and exported with the setup.py requirements. 

Also after removing `mhs-common` the tests all seem to run fine so I guess we didn't need the self-dependancy